src/format/strftime.rs

```diff
@@ -68,7 +68,7 @@ The following specifiers are available both to formatting and parsing.
 | `%r`  | `12:34:60 AM` | Hour-minute-second format in 12-hour clocks. Same as `%I:%M:%S %p`.   |
 |       |          |                                                                            |
 |       |          | **TIME ZONE SPECIFIERS:**                                                  |
-| `%Z`  | `ACST`   | Local time zone name. Skips all non-whitespace characters during parsing. [^8] |
+| `%Z`  | `ACST`   | Local time zone name. Skips all non-whitespace characters during parsing. Identical to `%:z` when formatting. [^8] |
```

Comment:
Maybe this could say something like:
"Time zone name. Depends on the `Display` implementation of the type implementing `TimeZone`. Skips all non-whitespace characters during parsing.

Reply:
Yeah, that would probably be an improvement.